script.js

```diff
@@ -0,0 +1,256 @@
+      } else if (element.innerText == 'Enter') {
+        output += '\n';
+      } else if (element.innerText == '') {
+        output += ' ';
+      } else if (element.innerText == 'Tab') {
+        output += '   ';
+      } else if (element.innerText == 'Caps Lock') {
+        output += '';
+      } else {
+        output += element.innerText;
+      }
+      document.querySelector('textarea').value = output;
+    } else {
+      if (event.code == "ShiftLeft") element[0].classList.add('keypress');
+      else if (event.code == "ShiftRight") element[1].classList.add('keypress');
+      else if (event.code == "ControlLeft") element[0].classList.add('keypress');
+      else if (event.code == "ControlRight") element[1].classList.add('keypress');
+      else if (event.code == "MetaLeft") element[0].classList.add('keypress');
+      else if (event.code == "MetaRight") element[1].classList.add('keypress');
+      else if (event.code == "AltLeft") element[0].classList.add('keypress');
+      else if (event.code == "AltRight") element[1].classList.add('keypress');
+    }
+    if (event.keyCode == 16) {
+      let langKeys;
+      isEng == 1 ? langKeys = engKeysShift : langKeys = rusKeysShift;
+      let i = 0;
+      let keys = document.querySelectorAll('#keyboard div');
+      keys.forEach((el) => {
+        el.innerText = langKeys[i];
+        i++;
+      });
+    }
+  }
+
+  document.addEventListener('keydown', (event) => {
+    keydown(event);
+  });
+  document.getElementById('keyboard').childNodes.forEach((e) => {
+    e.addEventListener('mousedown', (event) => {
+      let element = document.querySelectorAll('div[data="' + `${event.target.getAttribute('data')}` + '"]');
+      if (element.length == 1) {
+        element = element[0];
+        element.classList.add('keypress');
+      } else {
+        element = event.target;
+        element.classList.add('keypress');
+      }
+    });
+  })
+  document.getElementById('keyboard').childNodes.forEach((e) => {
+    e.addEventListener('mouseup', (event) => {
+     let element = document.querySelectorAll('div[data="' + `${event.target.getAttribute('data')}` + '"]');
+     if (element.length == 1) {
+       element = element[0];
+       element.classList.remove('keypress');
+       if (element.innerText == 'Backspace') {
+         output = output.substr(0, output.length-1);
+       } else if (element.innerText == 'Enter') {
+         output += '\n';
+       } else if (element.innerText == '') {
+         output += ' ';
+       } else if (element.innerText == 'Tab') {
+         output += '   ';
+       } else if (element.innerText == 'Caps Lock') {
+         output += '';
+       } else {
+         output += element.innerText;
+       }
+       document.querySelector('textarea').value = output;
+     } else {
+       element = event.target;
+       element.classList.remove('keypress');
+     }
+     let keyCode = event.target.getAttribute('data');
+     if (keyCode == 20) {
```

Comment:
Why `20`, write a comment what does it mean and check by `===`

Reply:
edited